src/UseCases/CancelDonation/CancelDonationUseCase.php

```diff
@@ -49,7 +58,7 @@ public function cancelDonation( CancelDonationRequest $cancellationRequest ): Ca
 
 		$this->sendConfirmationEmail( $donation );
```

Comment:
If something goes wrong when sending the mail, at the moment the user gets the error page instead of the cancellation result, while his donation is still canceled.